lib/utils/docker.coffee

```diff
@@ -156,15 +164,34 @@ exports.runBuild = (params, options, getBundleInfo, logStreams) ->
 	resolver = require('resin-bundle-resolve')
 	es = require('event-stream')
 	doodles = require('resin-doodles')
+	transpose = require('docker-qemu-transpose')
+	path = require('path')
 
 	logging = require('../utils/logging')
 
 	# The default build context is the current directory
 	params.source ?= '.'
 	logs = ''
+	# Only used in emulated builds
+	qemuPath = ''
+
+	Promise.try ->
+		return if not (options.emulated and platformNeedsQemu())
 
-	# Tar up the directory, ready for the build stream
-	tarDirectory(params.source)
+		hasQemu()
+		.then (present) ->
+			if !present
+				logging.logInfo(logStreams, 'Installing qemu for ARM emulation...')
+				installQemu()
+		.then ->
+			# Copy the qemu binary into the build context
+			copyQemu(params.source)
+		.then (binPath) ->
+			console.log(binPath)
```

Comment:
Oh yeah I accidentally left it in from when testing to answer your previous review.